app/javascript/forms/data-driven-form.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import FormRender from '@data-driven-forms/react-form-renderer';
+import { formFieldsMapper, layoutMapper } from '@data-driven-forms/pf3-component-mapper';
+
+const MiqFormRenderer = props => (
+  <FormRender
+    disableSubmit
```

Comment:
what's the purpose of always disabling submit?

Looks like submit is indeed always disabled. Doesn't sound like something we want :)